### PR TITLE
PLANNER-2823 Adapt VRP benchmarks to list variable

### PR DESCRIPTION
**Pull request by yurloc**
Depends on:
- https://github.com/kiegroup/optaplanner/pull/2233

### CODE REVIEW

File: optaplanner-benchmarks/optaplanner-perf-benchmark/src/main/java/org/jboss/qa/brms/performance/construction/vrp/VRPConstructionBenchmark.java [hunk 1]
```diff
@@ -14,9 +14,6 @@ public VRPConstructionBenchmark() {
         super(Examples.VEHICLE_ROUTING);
     }
 
-    @Param({"FIRST_FIT", "FIRST_FIT_DECREASING"})
```
Comment (yurloc) [inline optaplanner-benchmarks/optaplanner-perf-benchmark/src/main/java/org/jboss/qa/brms/performance/construction/vrp/VRPConstructionBenchmark.java:17]: List variable is incompatible with these construction heuristic algorithms. It only works with the default CH configuration (`<constructionHeuristic/>`).

File: optaplanner-benchmarks/optaplanner-perf-benchmark/src/main/java/org/jboss/qa/brms/performance/localsearch/vrp/moveselector/VRPSubChainChangeMoveSelectorBenchmark.java [hunk 2]
```diff
@@ -13,7 +13,7 @@ public class VRPSubChainChangeMoveSelectorBenchmark extends AbstractVRPMoveSelec
 
     @Override
     public List<MoveSelectorConfig> createMoveSelectorConfigList() {
-        return MoveSelectorConfigurations.createSubChainChangeMoveSelectorList();
+        return MoveSelectorConfigurations.createSubListChangeMoveSelectorList();
```
Comment (yurloc) [inline optaplanner-benchmarks/optaplanner-perf-benchmark/src/main/java/org/jboss/qa/brms/performance/localsearch/vrp/moveselector/VRPSubChainChangeMoveSelectorBenchmark.java:16]: @dupliaka The question is, should we also rename the benchmark class to `VRPSubListChangeMoveSelectorBenchmark.java`? We may lose continuity in Horreum reports. But of course, it doesn't make sense to have a `*SubChain*Benchmark` that doesn't test subChain move selectors but subList move selectors, that would be confusing.

Comment (dupliaka) [reply optaplanner-benchmarks/optaplanner-perf-benchmark/src/main/java/org/jboss/qa/brms/performance/localsearch/vrp/moveselector/VRPSubChainChangeMoveSelectorBenchmark.java:16]: Lets change it

File: optaplanner-benchmarks/optaplanner-perf-benchmark/src/main/java/org/jboss/qa/brms/performance/construction/vrp/VRPConstructionBenchmark.java (hunk 1 above)
Comment (dupliaka) [reply optaplanner-benchmarks/optaplanner-perf-benchmark/src/main/java/org/jboss/qa/brms/performance/construction/vrp/VRPConstructionBenchmark.java:17]: But I thought FIRST_FIT is actually a default for  constructionHeuristic

**Pull request merged**